tests/plugins/keyword_test.py

```diff
@@ -329,3 +369,19 @@ def test_analyze_example_negatives(self, file_content):
             # Make it start with `<`, (and end with `>`) so it hits our false-positive check
             line=file_content.replace('m{', '<').replace('}', '>'),
         )
+
+    @pytest.mark.skip(
+        reason='TODO: false positive heuristics need to be migrated over to filters/*',
+    )
```

Comment:
Did you mean to skip this test? All filters have been migrated already.

Reply:
I'm sorry, when I have implemented this feature, the filters haven't been migrated already. I will commit it comming soon.

Reply:
Hi @domanchi, I have reviewed the default filters of detect-secrets:

```JSON
"filters_used": [
    {
      "path": "detect_secrets.filters.allowlist.is_line_allowlisted"
    },
    {
      "path": "detect_secrets.filters.common.is_ignored_due_to_verification_policies",
      "min_level": 2
    },
    {
      "path": "detect_secrets.filters.heuristic.is_likely_id_string"
    },
    {
      "path": "detect_secrets.filters.heuristic.is_potential_uuid"
    },
    {
      "path": "detect_secrets.filters.heuristic.is_sequential_string"
    }
  ],
```
In the default filters, we haven't found any adequate to our test's needs. I think that the most useful filter in this case, can be `heuristic.is_templated_secret`. We haven't found any references to this filter in the [documentation](https://github.com/Yelp/detect-secrets/blob/pre-v1-launch/docs/filters.md#user-content-built-in-filters) and we think that it should be established by default. Are this tasks in your roadmap?

Reply:
Good catch. I must have missed this during my migration effort. I'll cut a ticket for this.

Reply:
https://github.com/Yelp/detect-secrets/pull/405